bigquery/google/cloud/bigquery/job.py

```diff
@@ -90,46 +90,94 @@ def _error_result_to_exception(error_result):
 
 
 class Compression(_EnumApiResourceProperty):
-    """Pseudo-enum for ``compression`` properties."""
+    """The compression type to use for exported files.
+
+    Possible values include `GZIP` and `NONE`. The default value is `NONE`.
+    """
     GZIP = 'GZIP'
     NONE = 'NONE'
 
 
 class CreateDisposition(_EnumApiResourceProperty):
-    """Pseudo-enum for ``create_disposition`` properties."""
+    """Specifies whether the job is allowed to create new tables.
+
+    The following values are supported:
+    `CREATE_IF_NEEDED`: If the table does not exist, BigQuery creates
+    the table.
+    `CREATE_NEVER`: The table must already exist. If it does not,
+    a 'notFound' error is returned in the job result.
+    The default value is `CREATE_IF_NEEDED`.
+
+    Creation, truncation and append actions occur as one atomic update
+    upon job completion.
+    """
     CREATE_IF_NEEDED = 'CREATE_IF_NEEDED'
     CREATE_NEVER = 'CREATE_NEVER'
 
 
 class DestinationFormat(_EnumApiResourceProperty):
-    """Pseudo-enum for ``destination_format`` properties."""
+    """The exported file format.
+
+    Possible values include `CSV`, `NEWLINE_DELIMITED_JSON` and `AVRO`.
+    The default value is `CSV`. Tables with nested or repeated fields
+    cannot be exported as CSV.
+    """
     CSV = 'CSV'
     NEWLINE_DELIMITED_JSON = 'NEWLINE_DELIMITED_JSON'
     AVRO = 'AVRO'
 
 
 class Encoding(_EnumApiResourceProperty):
-    """Pseudo-enum for ``encoding`` properties."""
+    """The character encoding of the data. The supported values
+    are `UTF-8` or `ISO-8859-1`. The default value is `UTF-8`.
```

Comment:
Those are the values, but not the names of the properties. I'd recommend making a list of the constants with their corresponding values for this one.